scripts/processing_manifest.py

```diff
@@ -46,11 +43,22 @@ class DatasetStatus(AindModel):
 
 
 class DataDescription(AindModel):
```

Comment:
It's not entirely clear to me why DataDescription is being overwritten here instead of just being imported. It can potentially cause conflicts downstream.

Reply:
Mostly because we just need John to provide some metadata directly from the rig that will be used to complete the actual metadata in the data description class in aind-data-schema.

Reply:
Since it's already in the main branch, we can probably open a separate thread on this. It's confusing because there's already a DataDescription class in aind_data_schema.